Jenkinsfile

```diff
@@ -44,6 +46,62 @@ pipeline {
                 }
             }
         }
+        stage('Build container serving the app [ PR ]') {
+            when {
+                changeRequest()
+            }
+            environment {
+                TAG = "PR-${CHANGE_ID}-${BUILD_NUMBER}"
+                DOCKER_CONFIG="/root/.docker"
+                LOCAL_REPOSITORY = "${LOCAL_REGISTRY}/molgenis/molgenis-app-lifelines-webshop"
+            }
+            steps {
+                container (name: 'kaniko', shell: '/busybox/sh') {
+                    sh "#!/busybox/sh\nmkdir -p ${DOCKER_CONFIG}"
+                    sh "#!/busybox/sh\necho '{\"auths\": {\"registry.molgenis.org\": {\"auth\": \"${NEXUS_AUTH}\"}}}' > ${DOCKER_CONFIG}/config.json"
+                    sh "#!/busybox/sh\n. ${WORKSPACE}/copy_dist_dir.sh"
+                    sh "#!/busybox/sh\n/kaniko/executor --context ${WORKSPACE}/docker --destination ${LOCAL_REPOSITORY}:${TAG}"
+                }
+            }
+        }
+        // TODO: deploy!
```

Comment:
please remove commented out code , and todo , we can just make a issue for the todo

Reply:
is work in progress